Libevent depends on OpenSSL

Libevent has a missing dependency, OpenSSL.
This may be platform specific.

uname -a
`Linux localhost 3.10.18 #1 SMP Sat Apr 9 17:50:02 PDT 2016 x86_64 Intel(R) Celeron(R) CPU N2840 @ 2.16GHz GenuineIntel GNU/Linux`

version
`7834.70.0 (Official Build) stable-channel swanky `

How to reproduce:
* install chromebrew
* run `crew install libevent`

```
Found libevent, version 2.0.21
Found libevent, version 2.0.21
No precompiled binary available for your platform, downloading source...
WARNING: timestamping does nothing in combination with -O. See the manual
for details.

--2016-04-27 16:50:40--  https://github.com/downloads/libevent/libevent/libevent-2.0.21-stable.tar.gz
Resolving github.com... 192.30.252.128
Connecting to github.com|192.30.252.128|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://cloud.github.com/downloads/libevent/libevent/libevent-2.0.21-stable.tar.gz [following]
--2016-04-27 16:50:41--  https://cloud.github.com/downloads/libevent/libevent/libevent-2.0.21-stable.tar.gz
Resolving cloud.github.com... 54.240.184.63, 54.240.184.170, 54.240.184.96, ...
Connecting to cloud.github.com|54.240.184.63|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 850772 (831K) [application/gzip]
--2016-04-27 16:50:41--  https://cloud.github.com/downloads/libevent/libevent/libevent-2.0.21-stable.tar.gz
Reusing existing connection to cloud.github.com:443.
HTTP request sent, awaiting response... 200 OK
Length: 850772 (831K) [application/gzip]
Saving to: 'libevent-2.0.21-stable.tar.gz'

     0K .......... .......... .......... .......... ..........  6%  747K 1s
    50K .......... .......... .......... .......... .......... 12% 1.11M 1s
   100K .......... .......... .......... .......... .......... 18% 1.49M 1s
   150K .......... .......... .......... .......... .......... 24% 1.22M 1s
   200K .......... .......... .......... .......... .......... 30% 1.88M 0s
   250K .......... .......... .......... .......... .......... 36% 2.16M 0s
   300K .......... .......... .......... .......... .......... 42% 2.71M 0s
   350K .......... .......... .......... .......... .......... 48% 3.40M 0s
   400K .......... .......... .......... .......... .......... 54% 3.27M 0s
   450K .......... .......... .......... .......... .......... 60% 1.80M 0s
   500K .......... .......... .......... .......... .......... 66% 2.98M 0s
   550K .......... .......... .......... .......... .......... 72% 3.08M 0s
   600K .......... .......... .......... .......... .......... 78% 2.47M 0s
   650K .......... .......... .......... .......... .......... 84% 3.09M 0s
   700K .......... .......... .......... .......... .......... 90% 2.71M 0s
   750K .......... .......... .......... .......... .......... 96% 2.63M 0s
   800K .......... .......... ..........                      100% 2.23M=0.4s

2016-04-27 16:50:41 (1.91 MB/s) - 'libevent-2.0.21-stable.tar.gz' saved [850772/850772]

Archive downloaded
Unpacking archive, this may take a while...
gcc (GCC) 4.8.1
Copyright (C) 2013 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Building from source, this may take a while...
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... no
checking for mawk... mawk
checking whether make sets $(MAKE)... yes
checking build system type... x86_64-unknown-linux-gnu
checking host system type... x86_64-unknown-linux-gnu
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking for style of include used by make... GNU
checking dependency style of gcc... gcc3
checking whether gcc and cc understand -c and -o together... yes
checking for a sed that does not truncate output... /bin/sed
checking whether ln -s works... yes
checking how to run the C preprocessor... gcc -E
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking whether gcc needs -traditional... no
checking how to print strings... printf
checking for a sed that does not truncate output... (cached) /bin/sed
checking for fgrep... /bin/grep -F
checking for ld used by gcc... /usr/local/x86_64-unknown-linux-gnu/bin/ld
checking if the linker (/usr/local/x86_64-unknown-linux-gnu/bin/ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... /usr/local/bin/nm -B
checking the name lister (/usr/local/bin/nm -B) interface... BSD nm
checking the maximum length of command line arguments... 1572864
checking whether the shell understands some XSI constructs... yes
checking whether the shell understands "+="... yes
checking how to convert x86_64-unknown-linux-gnu file names to x86_64-unknown-linux-gnu format... func_convert_file_noop
checking how to convert x86_64-unknown-linux-gnu file names to toolchain format... func_convert_file_noop
checking for /usr/local/x86_64-unknown-linux-gnu/bin/ld option to reload object files... -r
checking for objdump... objdump
checking how to recognize dependent libraries... pass_all
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for ar... ar
checking for archiver @FILE support... @
checking for strip... strip
checking for ranlib... ranlib
checking command to parse /usr/local/bin/nm -B output from gcc object... ok
checking for sysroot... no
./configure: line 6972: /usr/bin/file: No such file or directory
checking for mt... no
checking if : is a manifest tool... no
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if gcc supports -fno-rtti -fno-exceptions... ./configure: line 8498: diff: command not found
no
checking for gcc option to produce PIC... -fPIC -DPIC
checking if gcc PIC flag -fPIC -DPIC works... yes
checking if gcc static flag -static works... yes
checking if gcc supports -c -o file.o... yes
checking if gcc supports -c -o file.o... (cached) yes
checking whether the gcc linker (/usr/local/x86_64-unknown-linux-gnu/bin/ld) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... yes
checking for library containing inet_ntoa... none required
checking for library containing socket... none required
checking for library containing inet_aton... none required
checking for library containing clock_gettime... none required
checking for library containing sendfile... none required
checking for WIN32... no
checking for CYGWIN... no
checking zlib.h usability... yes
checking zlib.h presence... yes
checking for zlib.h... yes
checking for library containing inflateEnd... -lz
checking openssl/bio.h usability... no
checking openssl/bio.h presence... no
checking for openssl/bio.h... no
checking for library containing SSL_new... -lssl
checking for ANSI C header files... (cached) yes
checking fcntl.h usability... yes
checking fcntl.h presence... yes
checking for fcntl.h... yes
checking stdarg.h usability... yes
checking stdarg.h presence... yes
checking for stdarg.h... yes
checking for inttypes.h... (cached) yes
checking for stdint.h... (cached) yes
checking stddef.h usability... yes
checking stddef.h presence... yes
checking for stddef.h... yes
checking poll.h usability... yes
checking poll.h presence... yes
checking for poll.h... yes
checking for unistd.h... (cached) yes
checking sys/epoll.h usability... yes
checking sys/epoll.h presence... yes
checking for sys/epoll.h... yes
checking sys/time.h usability... yes
checking sys/time.h presence... yes
checking for sys/time.h... yes
checking sys/queue.h usability... yes
checking sys/queue.h presence... yes
checking for sys/queue.h... yes
checking sys/event.h usability... no
checking sys/event.h presence... no
checking for sys/event.h... no
checking sys/param.h usability... yes
checking sys/param.h presence... yes
checking for sys/param.h... yes
checking sys/ioctl.h usability... yes
checking sys/ioctl.h presence... yes
checking for sys/ioctl.h... yes
checking sys/select.h usability... yes
checking sys/select.h presence... yes
checking for sys/select.h... yes
checking sys/devpoll.h usability... no
checking sys/devpoll.h presence... no
checking for sys/devpoll.h... no
checking port.h usability... no
checking port.h presence... no
checking for port.h... no
checking netinet/in.h usability... yes
checking netinet/in.h presence... yes
checking for netinet/in.h... yes
checking netinet/in6.h usability... no
checking netinet/in6.h presence... no
checking for netinet/in6.h... no
checking sys/socket.h usability... yes
checking sys/socket.h presence... yes
checking for sys/socket.h... yes
checking sys/uio.h usability... yes
checking sys/uio.h presence... yes
checking for sys/uio.h... yes
checking arpa/inet.h usability... yes
checking arpa/inet.h presence... yes
checking for arpa/inet.h... yes
checking sys/eventfd.h usability... yes
checking sys/eventfd.h presence... yes
checking for sys/eventfd.h... yes
checking sys/mman.h usability... yes
checking sys/mman.h presence... yes
checking for sys/mman.h... yes
checking sys/sendfile.h usability... yes
checking sys/sendfile.h presence... yes
checking for sys/sendfile.h... yes
checking sys/wait.h usability... yes
checking sys/wait.h presence... yes
checking for sys/wait.h... yes
checking netdb.h usability... yes
checking netdb.h presence... yes
checking for netdb.h... yes
checking for sys/stat.h... (cached) yes
checking for sys/sysctl.h... yes
checking for TAILQ_FOREACH in sys/queue.h... yes
checking for timeradd in sys/time.h... yes
checking for timercmp in sys/time.h... yes
checking for timerclear in sys/time.h... yes
checking for timerisset in sys/time.h... yes
checking whether CTL_KERN is declared... yes
checking whether KERN_RANDOM is declared... yes
checking whether RANDOM_UUID is declared... yes
checking whether KERN_ARND is declared... no
checking for an ANSI C-conforming const... yes
checking for inline... inline
checking whether time.h and sys/time.h may both be included... yes
checking for gettimeofday... yes
checking for vasprintf... yes
checking for fcntl... yes
checking for clock_gettime... yes
checking for strtok_r... yes
checking for strsep... yes
checking for getnameinfo... yes
checking for strlcpy... no
checking for inet_ntop... yes
checking for inet_pton... yes
checking for signal... yes
checking for sigaction... yes
checking for strtoll... yes
checking for inet_aton... yes
checking for pipe... yes
checking for eventfd... yes
checking for sendfile... yes
checking for mmap... yes
checking for splice... yes
checking for arc4random... no
checking for arc4random_buf... no
checking for issetugid... no
checking for geteuid... yes
checking for getegid... yes
checking for getprotobynumber... yes
checking for setenv... yes
checking for unsetenv... yes
checking for putenv... yes
checking for sysctl... yes
checking for umask... yes
checking for getaddrinfo... yes
checking size of long... 8
checking for F_SETFD in fcntl.h... yes
checking for select... yes
checking for poll... yes
checking for epoll_ctl... yes
checking for port_create... no
checking for pid_t... yes
checking for size_t... yes
checking for ssize_t... yes
checking for uint64_t... yes
checking for uint32_t... yes
checking for uint16_t... yes
checking for uint8_t... yes
checking for uintptr_t... yes
checking for fd_mask... yes
checking size of long long... 8
checking size of long... (cached) 8
checking size of int... 4
checking size of short... 2
checking size of size_t... 8
checking size of void *... 8
checking for struct in6_addr... yes
checking for struct sockaddr_in6... yes
checking for sa_family_t... yes
checking for struct addrinfo... yes
checking for struct sockaddr_storage... yes
checking for struct in6_addr.s6_addr32... yes
checking for struct in6_addr.s6_addr16... yes
checking for struct sockaddr_in.sin_len... no
checking for struct sockaddr_in6.sin6_len... no
checking for struct sockaddr_storage.ss_family... yes
checking for struct sockaddr_storage.__ss_family... no
checking for socklen_t... yes
checking whether our compiler supports __func__... yes
checking for the pthreads library -lpthreads... no
checking whether pthreads work without any flags... no
checking whether pthreads work with -Kthread... no
checking whether pthreads work with -kthread... no
checking for the pthreads library -llthread... no
checking whether pthreads work with -pthread... yes
checking for joinable pthread attribute... PTHREAD_CREATE_JOINABLE
checking if more special flags are required for pthreads... no
checking size of pthread_t... 8
configure: creating ./config.status
config.status: creating libevent.pc
config.status: creating libevent_openssl.pc
config.status: creating libevent_pthreads.pc
config.status: creating Makefile
config.status: creating include/Makefile
config.status: creating test/Makefile
config.status: creating sample/Makefile
config.status: creating config.h
config.status: executing depfiles commands
config.status: executing libtool commands
test -d include/event2 || /bin/mkdir -p include/event2
/bin/sed -f ./make-event-config.sed < config.h > include/event2/event-config.hT
mv -f include/event2/event-config.hT include/event2/event-config.h
make  all-recursive
make[1]: Entering directory `/usr/local/tmp/crew/libevent-2.0.21-stable'
Making all in .
make[2]: Entering directory `/usr/local/tmp/crew/libevent-2.0.21-stable'
/bin/bash ./libtool --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -I./compat -I./include -I./include    -g -O2 -Wall -fno-strict-aliasing -pthread -MT event.lo -MD -MP -MF .deps/event.Tpo -c -o event.lo event.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT event.lo -MD -MP -MF .deps/event.Tpo -c event.c  -fPIC -DPIC -o .libs/event.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT event.lo -MD -MP -MF .deps/event.Tpo -c event.c -o event.o >/dev/null 2>&1
mv -f .deps/event.Tpo .deps/event.Plo
/bin/bash ./libtool --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -I./compat -I./include -I./include    -g -O2 -Wall -fno-strict-aliasing -pthread -MT evthread.lo -MD -MP -MF .deps/evthread.Tpo -c -o evthread.lo evthread.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT evthread.lo -MD -MP -MF .deps/evthread.Tpo -c evthread.c  -fPIC -DPIC -o .libs/evthread.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT evthread.lo -MD -MP -MF .deps/evthread.Tpo -c evthread.c -o evthread.o >/dev/null 2>&1
mv -f .deps/evthread.Tpo .deps/evthread.Plo
/bin/bash ./libtool --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -I./compat -I./include -I./include    -g -O2 -Wall -fno-strict-aliasing -pthread -MT buffer.lo -MD -MP -MF .deps/buffer.Tpo -c -o buffer.lo buffer.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT buffer.lo -MD -MP -MF .deps/buffer.Tpo -c buffer.c  -fPIC -DPIC -o .libs/buffer.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT buffer.lo -MD -MP -MF .deps/buffer.Tpo -c buffer.c -o buffer.o >/dev/null 2>&1
mv -f .deps/buffer.Tpo .deps/buffer.Plo
/bin/bash ./libtool --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -I./compat -I./include -I./include    -g -O2 -Wall -fno-strict-aliasing -pthread -MT bufferevent.lo -MD -MP -MF .deps/bufferevent.Tpo -c -o bufferevent.lo bufferevent.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT bufferevent.lo -MD -MP -MF .deps/bufferevent.Tpo -c bufferevent.c  -fPIC -DPIC -o .libs/bufferevent.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT bufferevent.lo -MD -MP -MF .deps/bufferevent.Tpo -c bufferevent.c -o bufferevent.o >/dev/null 2>&1
mv -f .deps/bufferevent.Tpo .deps/bufferevent.Plo
/bin/bash ./libtool --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -I./compat -I./include -I./include    -g -O2 -Wall -fno-strict-aliasing -pthread -MT bufferevent_sock.lo -MD -MP -MF .deps/bufferevent_sock.Tpo -c -o bufferevent_sock.lo bufferevent_sock.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT bufferevent_sock.lo -MD -MP -MF .deps/bufferevent_sock.Tpo -c bufferevent_sock.c  -fPIC -DPIC -o .libs/bufferevent_sock.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT bufferevent_sock.lo -MD -MP -MF .deps/bufferevent_sock.Tpo -c bufferevent_sock.c -o bufferevent_sock.o >/dev/null 2>&1
mv -f .deps/bufferevent_sock.Tpo .deps/bufferevent_sock.Plo
/bin/bash ./libtool --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -I./compat -I./include -I./include    -g -O2 -Wall -fno-strict-aliasing -pthread -MT bufferevent_filter.lo -MD -MP -MF .deps/bufferevent_filter.Tpo -c -o bufferevent_filter.lo bufferevent_filter.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT bufferevent_filter.lo -MD -MP -MF .deps/bufferevent_filter.Tpo -c bufferevent_filter.c  -fPIC -DPIC -o .libs/bufferevent_filter.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT bufferevent_filter.lo -MD -MP -MF .deps/bufferevent_filter.Tpo -c bufferevent_filter.c -o bufferevent_filter.o >/dev/null 2>&1
mv -f .deps/bufferevent_filter.Tpo .deps/bufferevent_filter.Plo
/bin/bash ./libtool --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -I./compat -I./include -I./include    -g -O2 -Wall -fno-strict-aliasing -pthread -MT bufferevent_pair.lo -MD -MP -MF .deps/bufferevent_pair.Tpo -c -o bufferevent_pair.lo bufferevent_pair.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT bufferevent_pair.lo -MD -MP -MF .deps/bufferevent_pair.Tpo -c bufferevent_pair.c  -fPIC -DPIC -o .libs/bufferevent_pair.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT bufferevent_pair.lo -MD -MP -MF .deps/bufferevent_pair.Tpo -c bufferevent_pair.c -o bufferevent_pair.o >/dev/null 2>&1
mv -f .deps/bufferevent_pair.Tpo .deps/bufferevent_pair.Plo
/bin/bash ./libtool --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -I./compat -I./include -I./include    -g -O2 -Wall -fno-strict-aliasing -pthread -MT listener.lo -MD -MP -MF .deps/listener.Tpo -c -o listener.lo listener.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT listener.lo -MD -MP -MF .deps/listener.Tpo -c listener.c  -fPIC -DPIC -o .libs/listener.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT listener.lo -MD -MP -MF .deps/listener.Tpo -c listener.c -o listener.o >/dev/null 2>&1
mv -f .deps/listener.Tpo .deps/listener.Plo
/bin/bash ./libtool --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -I./compat -I./include -I./include    -g -O2 -Wall -fno-strict-aliasing -pthread -MT bufferevent_ratelim.lo -MD -MP -MF .deps/bufferevent_ratelim.Tpo -c -o bufferevent_ratelim.lo bufferevent_ratelim.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT bufferevent_ratelim.lo -MD -MP -MF .deps/bufferevent_ratelim.Tpo -c bufferevent_ratelim.c  -fPIC -DPIC -o .libs/bufferevent_ratelim.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT bufferevent_ratelim.lo -MD -MP -MF .deps/bufferevent_ratelim.Tpo -c bufferevent_ratelim.c -o bufferevent_ratelim.o >/dev/null 2>&1
mv -f .deps/bufferevent_ratelim.Tpo .deps/bufferevent_ratelim.Plo
/bin/bash ./libtool --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -I./compat -I./include -I./include    -g -O2 -Wall -fno-strict-aliasing -pthread -MT evmap.lo -MD -MP -MF .deps/evmap.Tpo -c -o evmap.lo evmap.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT evmap.lo -MD -MP -MF .deps/evmap.Tpo -c evmap.c  -fPIC -DPIC -o .libs/evmap.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT evmap.lo -MD -MP -MF .deps/evmap.Tpo -c evmap.c -o evmap.o >/dev/null 2>&1
mv -f .deps/evmap.Tpo .deps/evmap.Plo
/bin/bash ./libtool --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -I./compat -I./include -I./include    -g -O2 -Wall -fno-strict-aliasing -pthread -MT log.lo -MD -MP -MF .deps/log.Tpo -c -o log.lo log.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT log.lo -MD -MP -MF .deps/log.Tpo -c log.c  -fPIC -DPIC -o .libs/log.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT log.lo -MD -MP -MF .deps/log.Tpo -c log.c -o log.o >/dev/null 2>&1
mv -f .deps/log.Tpo .deps/log.Plo
/bin/bash ./libtool --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -I./compat -I./include -I./include    -g -O2 -Wall -fno-strict-aliasing -pthread -MT evutil.lo -MD -MP -MF .deps/evutil.Tpo -c -o evutil.lo evutil.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT evutil.lo -MD -MP -MF .deps/evutil.Tpo -c evutil.c  -fPIC -DPIC -o .libs/evutil.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT evutil.lo -MD -MP -MF .deps/evutil.Tpo -c evutil.c -o evutil.o >/dev/null 2>&1
mv -f .deps/evutil.Tpo .deps/evutil.Plo
/bin/bash ./libtool --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -I./compat -I./include -I./include    -g -O2 -Wall -fno-strict-aliasing -pthread -MT evutil_rand.lo -MD -MP -MF .deps/evutil_rand.Tpo -c -o evutil_rand.lo evutil_rand.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT evutil_rand.lo -MD -MP -MF .deps/evutil_rand.Tpo -c evutil_rand.c  -fPIC -DPIC -o .libs/evutil_rand.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT evutil_rand.lo -MD -MP -MF .deps/evutil_rand.Tpo -c evutil_rand.c -o evutil_rand.o >/dev/null 2>&1
mv -f .deps/evutil_rand.Tpo .deps/evutil_rand.Plo
/bin/bash ./libtool --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -I./compat -I./include -I./include    -g -O2 -Wall -fno-strict-aliasing -pthread -MT strlcpy.lo -MD -MP -MF .deps/strlcpy.Tpo -c -o strlcpy.lo strlcpy.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT strlcpy.lo -MD -MP -MF .deps/strlcpy.Tpo -c strlcpy.c  -fPIC -DPIC -o .libs/strlcpy.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT strlcpy.lo -MD -MP -MF .deps/strlcpy.Tpo -c strlcpy.c -o strlcpy.o >/dev/null 2>&1
mv -f .deps/strlcpy.Tpo .deps/strlcpy.Plo
/bin/bash ./libtool --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -I./compat -I./include -I./include    -g -O2 -Wall -fno-strict-aliasing -pthread -MT select.lo -MD -MP -MF .deps/select.Tpo -c -o select.lo select.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT select.lo -MD -MP -MF .deps/select.Tpo -c select.c  -fPIC -DPIC -o .libs/select.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT select.lo -MD -MP -MF .deps/select.Tpo -c select.c -o select.o >/dev/null 2>&1
mv -f .deps/select.Tpo .deps/select.Plo
/bin/bash ./libtool --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -I./compat -I./include -I./include    -g -O2 -Wall -fno-strict-aliasing -pthread -MT poll.lo -MD -MP -MF .deps/poll.Tpo -c -o poll.lo poll.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT poll.lo -MD -MP -MF .deps/poll.Tpo -c poll.c  -fPIC -DPIC -o .libs/poll.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT poll.lo -MD -MP -MF .deps/poll.Tpo -c poll.c -o poll.o >/dev/null 2>&1
mv -f .deps/poll.Tpo .deps/poll.Plo
/bin/bash ./libtool --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -I./compat -I./include -I./include    -g -O2 -Wall -fno-strict-aliasing -pthread -MT epoll.lo -MD -MP -MF .deps/epoll.Tpo -c -o epoll.lo epoll.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT epoll.lo -MD -MP -MF .deps/epoll.Tpo -c epoll.c  -fPIC -DPIC -o .libs/epoll.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT epoll.lo -MD -MP -MF .deps/epoll.Tpo -c epoll.c -o epoll.o >/dev/null 2>&1
mv -f .deps/epoll.Tpo .deps/epoll.Plo
/bin/bash ./libtool --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -I./compat -I./include -I./include    -g -O2 -Wall -fno-strict-aliasing -pthread -MT signal.lo -MD -MP -MF .deps/signal.Tpo -c -o signal.lo signal.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT signal.lo -MD -MP -MF .deps/signal.Tpo -c signal.c  -fPIC -DPIC -o .libs/signal.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT signal.lo -MD -MP -MF .deps/signal.Tpo -c signal.c -o signal.o >/dev/null 2>&1
mv -f .deps/signal.Tpo .deps/signal.Plo
/bin/bash ./libtool --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -I./compat -I./include -I./include    -g -O2 -Wall -fno-strict-aliasing -pthread -MT event_tagging.lo -MD -MP -MF .deps/event_tagging.Tpo -c -o event_tagging.lo event_tagging.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT event_tagging.lo -MD -MP -MF .deps/event_tagging.Tpo -c event_tagging.c  -fPIC -DPIC -o .libs/event_tagging.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT event_tagging.lo -MD -MP -MF .deps/event_tagging.Tpo -c event_tagging.c -o event_tagging.o >/dev/null 2>&1
mv -f .deps/event_tagging.Tpo .deps/event_tagging.Plo
/bin/bash ./libtool --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -I./compat -I./include -I./include    -g -O2 -Wall -fno-strict-aliasing -pthread -MT http.lo -MD -MP -MF .deps/http.Tpo -c -o http.lo http.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT http.lo -MD -MP -MF .deps/http.Tpo -c http.c  -fPIC -DPIC -o .libs/http.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT http.lo -MD -MP -MF .deps/http.Tpo -c http.c -o http.o >/dev/null 2>&1
mv -f .deps/http.Tpo .deps/http.Plo
/bin/bash ./libtool --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -I./compat -I./include -I./include    -g -O2 -Wall -fno-strict-aliasing -pthread -MT evdns.lo -MD -MP -MF .deps/evdns.Tpo -c -o evdns.lo evdns.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT evdns.lo -MD -MP -MF .deps/evdns.Tpo -c evdns.c  -fPIC -DPIC -o .libs/evdns.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT evdns.lo -MD -MP -MF .deps/evdns.Tpo -c evdns.c -o evdns.o >/dev/null 2>&1
mv -f .deps/evdns.Tpo .deps/evdns.Plo
/bin/bash ./libtool --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -I./compat -I./include -I./include    -g -O2 -Wall -fno-strict-aliasing -pthread -MT evrpc.lo -MD -MP -MF .deps/evrpc.Tpo -c -o evrpc.lo evrpc.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT evrpc.lo -MD -MP -MF .deps/evrpc.Tpo -c evrpc.c  -fPIC -DPIC -o .libs/evrpc.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT evrpc.lo -MD -MP -MF .deps/evrpc.Tpo -c evrpc.c -o evrpc.o >/dev/null 2>&1
mv -f .deps/evrpc.Tpo .deps/evrpc.Plo
/bin/bash ./libtool --tag=CC   --mode=link gcc  -g -O2 -Wall -fno-strict-aliasing -pthread -version-info 6:9:1 -release 2.0   -o libevent.la -rpath /usr/local/lib event.lo evthread.lo buffer.lo bufferevent.lo bufferevent_sock.lo bufferevent_filter.lo bufferevent_pair.lo listener.lo bufferevent_ratelim.lo evmap.lo log.lo evutil.lo evutil_rand.lo strlcpy.lo select.lo poll.lo   epoll.lo  signal.lo event_tagging.lo http.lo evdns.lo evrpc.lo  
libtool: link: gcc -shared  -fPIC -DPIC  .libs/event.o .libs/evthread.o .libs/buffer.o .libs/bufferevent.o .libs/bufferevent_sock.o .libs/bufferevent_filter.o .libs/bufferevent_pair.o .libs/listener.o .libs/bufferevent_ratelim.o .libs/evmap.o .libs/log.o .libs/evutil.o .libs/evutil_rand.o .libs/strlcpy.o .libs/select.o .libs/poll.o .libs/epoll.o .libs/signal.o .libs/event_tagging.o .libs/http.o .libs/evdns.o .libs/evrpc.o    -O2 -pthread   -pthread -Wl,-soname -Wl,libevent-2.0.so.5 -o .libs/libevent-2.0.so.5.1.9
libtool: link: (cd ".libs" && rm -f "libevent-2.0.so.5" && ln -s "libevent-2.0.so.5.1.9" "libevent-2.0.so.5")
libtool: link: (cd ".libs" && rm -f "libevent.so" && ln -s "libevent-2.0.so.5.1.9" "libevent.so")
libtool: link: ar cru .libs/libevent.a  event.o evthread.o buffer.o bufferevent.o bufferevent_sock.o bufferevent_filter.o bufferevent_pair.o listener.o bufferevent_ratelim.o evmap.o log.o evutil.o evutil_rand.o strlcpy.o select.o poll.o epoll.o signal.o event_tagging.o http.o evdns.o evrpc.o
libtool: link: ranlib .libs/libevent.a
libtool: link: ( cd ".libs" && rm -f "libevent.la" && ln -s "../libevent.la" "libevent.la" )
/bin/bash ./libtool --tag=CC   --mode=link gcc  -g -O2 -Wall -fno-strict-aliasing -pthread -version-info 6:9:1 -release 2.0   -o libevent_core.la -rpath /usr/local/lib event.lo evthread.lo buffer.lo bufferevent.lo bufferevent_sock.lo bufferevent_filter.lo bufferevent_pair.lo listener.lo bufferevent_ratelim.lo evmap.lo log.lo evutil.lo evutil_rand.lo strlcpy.lo select.lo poll.lo   epoll.lo  signal.lo  
libtool: link: gcc -shared  -fPIC -DPIC  .libs/event.o .libs/evthread.o .libs/buffer.o .libs/bufferevent.o .libs/bufferevent_sock.o .libs/bufferevent_filter.o .libs/bufferevent_pair.o .libs/listener.o .libs/bufferevent_ratelim.o .libs/evmap.o .libs/log.o .libs/evutil.o .libs/evutil_rand.o .libs/strlcpy.o .libs/select.o .libs/poll.o .libs/epoll.o .libs/signal.o    -O2 -pthread   -pthread -Wl,-soname -Wl,libevent_core-2.0.so.5 -o .libs/libevent_core-2.0.so.5.1.9
libtool: link: (cd ".libs" && rm -f "libevent_core-2.0.so.5" && ln -s "libevent_core-2.0.so.5.1.9" "libevent_core-2.0.so.5")
libtool: link: (cd ".libs" && rm -f "libevent_core.so" && ln -s "libevent_core-2.0.so.5.1.9" "libevent_core.so")
libtool: link: ar cru .libs/libevent_core.a  event.o evthread.o buffer.o bufferevent.o bufferevent_sock.o bufferevent_filter.o bufferevent_pair.o listener.o bufferevent_ratelim.o evmap.o log.o evutil.o evutil_rand.o strlcpy.o select.o poll.o epoll.o signal.o
libtool: link: ranlib .libs/libevent_core.a
libtool: link: ( cd ".libs" && rm -f "libevent_core.la" && ln -s "../libevent_core.la" "libevent_core.la" )
/bin/bash ./libtool --tag=CC   --mode=link gcc  -g -O2 -Wall -fno-strict-aliasing -pthread -version-info 6:9:1 -release 2.0   -o libevent_extra.la -rpath /usr/local/lib event_tagging.lo http.lo evdns.lo evrpc.lo   
libtool: link: gcc -shared  -fPIC -DPIC  .libs/event_tagging.o .libs/http.o .libs/evdns.o .libs/evrpc.o    -O2 -pthread   -pthread -Wl,-soname -Wl,libevent_extra-2.0.so.5 -o .libs/libevent_extra-2.0.so.5.1.9
libtool: link: (cd ".libs" && rm -f "libevent_extra-2.0.so.5" && ln -s "libevent_extra-2.0.so.5.1.9" "libevent_extra-2.0.so.5")
libtool: link: (cd ".libs" && rm -f "libevent_extra.so" && ln -s "libevent_extra-2.0.so.5.1.9" "libevent_extra.so")
libtool: link: ar cru .libs/libevent_extra.a  event_tagging.o http.o evdns.o evrpc.o
libtool: link: ranlib .libs/libevent_extra.a
libtool: link: ( cd ".libs" && rm -f "libevent_extra.la" && ln -s "../libevent_extra.la" "libevent_extra.la" )
/bin/bash ./libtool --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -I./compat -I./include -I./include    -g -O2 -Wall -fno-strict-aliasing -pthread -MT evthread_pthread.lo -MD -MP -MF .deps/evthread_pthread.Tpo -c -o evthread_pthread.lo evthread_pthread.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT evthread_pthread.lo -MD -MP -MF .deps/evthread_pthread.Tpo -c evthread_pthread.c  -fPIC -DPIC -o .libs/evthread_pthread.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT evthread_pthread.lo -MD -MP -MF .deps/evthread_pthread.Tpo -c evthread_pthread.c -o evthread_pthread.o >/dev/null 2>&1
mv -f .deps/evthread_pthread.Tpo .deps/evthread_pthread.Plo
/bin/bash ./libtool --tag=CC   --mode=link gcc  -g -O2 -Wall -fno-strict-aliasing -pthread -version-info 6:9:1 -release 2.0   -o libevent_pthreads.la -rpath /usr/local/lib evthread_pthread.lo  
libtool: link: gcc -shared  -fPIC -DPIC  .libs/evthread_pthread.o    -O2 -pthread   -pthread -Wl,-soname -Wl,libevent_pthreads-2.0.so.5 -o .libs/libevent_pthreads-2.0.so.5.1.9
libtool: link: (cd ".libs" && rm -f "libevent_pthreads-2.0.so.5" && ln -s "libevent_pthreads-2.0.so.5.1.9" "libevent_pthreads-2.0.so.5")
libtool: link: (cd ".libs" && rm -f "libevent_pthreads.so" && ln -s "libevent_pthreads-2.0.so.5.1.9" "libevent_pthreads.so")
libtool: link: ar cru .libs/libevent_pthreads.a  evthread_pthread.o
libtool: link: ranlib .libs/libevent_pthreads.a
libtool: link: ( cd ".libs" && rm -f "libevent_pthreads.la" && ln -s "../libevent_pthreads.la" "libevent_pthreads.la" )
/bin/bash ./libtool --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.  -I./compat -I./include -I./include    -g -O2 -Wall -fno-strict-aliasing -pthread -MT bufferevent_openssl.lo -MD -MP -MF .deps/bufferevent_openssl.Tpo -c -o bufferevent_openssl.lo bufferevent_openssl.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./compat -I./include -I./include -g -O2 -Wall -fno-strict-aliasing -pthread -MT bufferevent_openssl.lo -MD -MP -MF .deps/bufferevent_openssl.Tpo -c bufferevent_openssl.c  -fPIC -DPIC -o .libs/bufferevent_openssl.o
bufferevent_openssl.c:60:25: fatal error: openssl/bio.h: No such file or directory
 #include <openssl/bio.h>
                         ^
compilation terminated.
make[2]: *** [bufferevent_openssl.lo] Error 1
make[2]: Leaving directory `/usr/local/tmp/crew/libevent-2.0.21-stable'
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory `/usr/local/tmp/crew/libevent-2.0.21-stable'
make: *** [all] Error 2
libevent failed to install: `make` exited with 2
```

--- a/crew
+++ b/crew
@@ -142,7 +142,7 @@ def download
     sha1sum = @pkg.binary_sha1[@device[:architecture]]
   end
   Dir.chdir CREW_BREW_DIR do
-    system('wget', '--continue', '--content-disposition', '--no-check-certificate', '-N', url, '-O', filename)
+    system('wget', '--continue', '--no-check-certificate', url, '-O', filename)
     abort 'Checksum mismatch :/ try again' unless Digest::SHA1.hexdigest( File.read("./#{filename}") ) == sha1sum
   end
   puts "Archive downloaded"

--- a/packages/libevent.rb
+++ b/packages/libevent.rb
@@ -1,12 +1,12 @@
 require 'package'                                                      	# include package class file
  
 class Libevent < Package                                            	# name the package and make it a Package class instance
-  version '2.0.21'                                               	                                      # software version
-  source_url 'https://github.com/downloads/libevent/libevent/libevent-2.0.21-stable.tar.gz'     # software source tarball url
-  source_sha1 '3e6674772eb77de24908c6267c698146420ab699'          	# source tarball sha1 sum
+  version '2.0.22'                                               	                                      # software version
+  source_url 'https://github.com/libevent/libevent/releases/download/release-2.0.22-stable/libevent-2.0.22-stable.tar.gz'     # software source tarball url
+  source_sha1 'a586882bc93a208318c70fc7077ed8fca9862864'          	# source tarball sha1 sum
   
   def self.build                                                  # self.build contains commands needed to build the software from source
-    system "./configure"
+    system "./configure --disable-openssl"
     system "make"                                                 # ordered chronologically
   end
   

--- a/packages/tmux.rb
+++ b/packages/tmux.rb
@@ -1,13 +1,12 @@
 require 'package'                                                      	# include package class file
  
 class Tmux < Package                                            	# name the package and make it a Package class instance
-  version '1.9a'                                               	                                      # software version
-  source_url 'http://downloads.sourceforge.net/tmux/tmux-1.9a.tar.gz'     # software source tarball url
-  source_sha1 '815264268e63c6c85fe8784e06a840883fcfc6a2'          	# source tarball sha1 sum
+  version '2.2'                                               	                                      # software version
+  source_url 'https://github.com/tmux/tmux/releases/download/2.2/tmux-2.2.tar.gz'     # software source tarball url
+  source_sha1 '5ed1430bc7ef44c227e64e9401c686573dd0791a'          	# source tarball sha1 sum
   
   depends_on 'readline'                                            # software dependencies
   depends_on 'libevent'
-  depends_on 'openssl'
   depends_on 'ncurses'
   
   def self.build                                                  # self.build contains commands needed to build the software from source

